### ISSUE TITLE
Basic component guide

### ISSUE THREAD

**Issue by amilton33** [orangeglo/PS2PSP#3]
Hello, I want to contribute to the project to add a blueretro directly to the ps2 controller port, would you have a guide with the name of the components used?

**Comment by luizoti** [comment]
Good idea, I suppose it is possible to integrate a small esp32 inside the psp, allowing mixed use when connecting to the tv and connecting the controller.

**Comment by Cyrus331** [comment]
I also thought of this idea, this project plus blueretro I think it can be realized，I hope orangeglo can make it happen ;)

**Comment by amilton33** [comment]
There's already one with bluetooth functionality, now the creator is adapting it for the psp 3000
https://github.com/ste2425/PSP-Consolizer

**Comment by Cyrus331** [comment]
Yes, I have also paid attention to the PSP-Consolizer project, but that project is more complicated, and the combination of blueretro and PS2PSP will be much simpler.

**Comment by amilton33** [comment]
But you don't need to do this project to use the blueretro later, just use an Esp-32 with the bluepad32 library and you will have the same result with the bluetooth option, or you can do this project and instead of putting a ps2 pad on the pcb, connect it directly to an esp-32 with a ps2 blueretro already installed on it.

**Comment by luizoti** [comment]
I think blueretro would bring more flexibility of controls, because it would be possible to connect any compatible with it.

**Comment by Bma36** [comment]
As I said before, to add the blueretro to this project just compile the esp-32 and solder the wires according to the PS2 blueretro schematic to the points on the pcb where the control connector goes

**Comment by Bma36** [comment]
As I said before, to add the blueretro to this project just compile the esp-32 and solder the wires according to the PS2 blueretro schematic to the points on the pcb where the control connector goes

**Comment by Cyrus331** [comment]
Have any friends realized this project? I am trying to make a PCB now, but I found that there seem to be two versions. The difference lies in the anlog, one with connectro and one without. Is the final version with connectro? Does this need to be linked? I can’t see it from the picture here[https://epbarger.github.io/assets/psp_consolized.jpg](url)

**Comment by epbarger** [comment]
The PS2 controller port connector is harvested from one of these cheap usb adapters - https://www.aliexpress.us/item/3256803457150606.html

**Comment by Bma36** [comment]
Remembering that this code is for the atmega 328PB, but changing the code you can compile it on the atmega328P, I will add an esp-32 with blueretro.

**Comment by davidxgames** [comment]
> Creo que blueretro traería más flexibilidad de controles, porque sería posible conectar cualquiera compatible con él.

In theory blueretro and Bluepad have the same control compatibility, something that can be added with bluepad is rumble every time you shoot in a game, and use the right stick to control the X OR square and triangle buttons, in FPS.

**Comment by davidxgames** [comment]
> Creo que blueretro traería más flexibilidad de controles, porque sería posible conectar cualquiera compatible con él.

In theory blueretro and Bluepad have the same control compatibility, something that can be added with bluepad is rumble every time you shoot in a game, and use the right stick to control the X OR square and triangle buttons, in FPS.

**Comment by Bma36** [comment]
> There's already one with bluetooth functionality, now the creator is adapting it for the psp 3000
> https://github.com/ste2425/PSP-Consolizer

The project with this pcb is great, include blueretro and still have the option to use dualshock with cable, for those who want an internal consolizer with esp-32, above there is the link of the developer Scoop that already made the project with an arduino nano 33

**Comment by Cyrus331** [comment]
> 请记住，此代码适用于 atmega 328PB，但更改代码后您可以在 atmega328P 上编译它，我将添加带有 blueretro 的 esp-32。

looking forward to your work

**Comment by amilton33** [comment]
I adapted the code to use an atmega 328P in an arduino nano, I am waiting for the apartment cables and fpcs ordered at aliexpress to arrive, as soon as I do the tests I will add blueretro so that it can have physical connection with ps2 control and wireless.

**Comment by VirtuaVespa** [comment]
> Have any friends realized this project? I am trying to make a PCB now, but I found that there seem to be two versions. The difference lies in the anlog, one with connectro and one without. Is the final version with connectro? Does this need to be linked? I can’t see it from the picture here[https://epbarger.github.io/assets/psp_consolized.jpg](url)

Hey where did you see the second PCB with the analog? the one on OSHpark is without

**Comment by VirtuaVespa** [comment]
> Have any friends realized this project? I am trying to make a PCB now, but I found that there seem to be two versions. The difference lies in the anlog, one with connectro and one without. Is the final version with connectro? Does this need to be linked? I can’t see it from the picture here[https://epbarger.github.io/assets/psp_consolized.jpg](url)

The PCB with the 'analog' connector is a forked version by DobaMuffin for the PSP3000 I believe.
